src/Sender.php

```diff
@@ -191,7 +191,7 @@ private function createRequestHeaderArray()
      * @param array $content
      * @return string json
      */
-    private function createRequestBodyJson($to, $content)
+    protected function createRequestBodyJson($to, $content)
```

Comment:
__[phpmd]__
* Avoid variables with short names like $to. Configured minimum length is 3.